README.md

```diff
@@ -111,87 +116,42 @@ workon cfgov-refresh
 cd cfgov-refresh
 
 # Index the latest content from the API output from a WordPress and Django back-end.
-# **This requires the constants in INSTALL - Configuration to be set**
+# **This requires the constants in INSTALL - Stand alone installation to be set**
 python cfgov/manage.py sheer_index
+```
 
-# From the Project root, start server.
-python cfgov/manage.py runserver
+> **NOTE:**
+  To view the indexed content you can use a tool called
+  [elasticsearch-head](http://mobz.github.io/elasticsearch-head/).
 
-# **Note**
-# If prompted to migrate database changes, stop the server ctrl+c and run these commands
-python cfgov/manage.py migrate
-./initial-data.sh
-./runserver.sh
+From the project root, start the Django server:
 
-# To set up a superuser in order to access the admin
-python cfgov/manage.py createsuperuser
+```bash
+./runserver.sh
 ```
 
-To view the site browse to: <http://localhost:8000>
-
-To view the project layout docs and pattern library,
-browse to <http://localhost:8000/docs>
-
-To view the indexed content you can use a tool called
-[elasticsearch-head](http://mobz.github.io/elasticsearch-head/).
+> **NOTE:** If prompted to migrate database changes,
+  stop the server with `ctrl` + `c` and run these commands:
+  ```
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2098/commits/2ecd3189c9096eda37a5ff143c907759e91cf1a4